.github/workflows/auto-pr-creation.yml

```
@@ -97,12 +114,22 @@ jobs:
 
           def find_epic_issue():
               """Find the epic issue"""
-              cmd = ['gh', 'issue', 'list', '--search', 'Epic:', '--json', 'number']
+              cmd = ['gh', 'issue', 'list', '--search', 'Epic:', '--state', 'open']
               
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)